docuware-connector-demo/src_hd/com/axonivy/connector/docuware/connector/demo/DocuWareDemoUI/DocuWareDemoUIProcess.p.json

```diff
@@ -0,0 +1,197 @@
+        }
+      },
+      "visual" : {
+        "at" : { "x" : 248, "y" : 256 }
+      },
+      "connect" : { "id" : "f10", "to" : "f9" }
+    }, {
+      "id" : "f9",
+      "type" : "RestClientCall",
+      "name" : "Query Documents",
+      "config" : {
+        "path" : "/FileCabinets/{FileCabinetId}/Documents",
+        "headers" : {
+          "Accept" : "application/xml"
+        },
+        "clientId" : "02d1eec1-32e9-4316-afc3-793448486203",
+        "clientErrorCode" : "ivy:error:rest:client",
+        "queryParams" : {
+          "q" : "",
+          "fields" : "",
+          "sortOrder" : ""
+        },
+        "statusErrorCode" : "ivy:error:rest:client",
+        "responseMapping" : {
+          "out.documents" : "result.items.getItem()"
+        },
+        "templateParams" : {
+          "FileCabinetId" : "in.fileCabinet.getId()"
+        },
+        "resultType" : "com.docuware.dev.schema._public.services.platform.DocumentsQueryResult"
+      },
+      "visual" : {
+        "at" : { "x" : 440, "y" : 256 }
+      },
+      "connect" : { "id" : "f12", "to" : "f11" }
+    }, {
+      "id" : "f11",
+      "type" : "HtmlDialogEnd",
+      "visual" : {
+        "at" : { "x" : 592, "y" : 256 }
+      }
+    }, {
+      "id" : "f13",
+      "type" : "HtmlDialogMethodStart",
+      "name" : "onClickEditDocument(String)",
+      "config" : {
+        "callSignature" : "onClickEditDocument",
+        "input" : {
+          "params" : [
+            { "name" : "documentId", "type" : "String" }
+          ],
+          "map" : {
+            "out.documentId" : "param.documentId"
+          }
+        },
+        "guid" : "1935FA55672C1004"
+      },
+      "visual" : {
+        "at" : { "x" : 104, "y" : 368 }
+      },
+      "connect" : { "id" : "f15", "to" : "f21" }
+    }, {
+      "id" : "f21",
+      "type" : "RestClientCall",
+      "name" : "Fetch Document Properties",
+      "config" : {
+        "path" : "/FileCabinets/{FileCabinetId}/Documents/{DocumentId}",
+        "headers" : {
+          "Accept" : "application/json"
+        },
+        "clientId" : "02d1eec1-32e9-4316-afc3-793448486203",
+        "clientErrorCode" : "ivy:error:rest:client",
+        "method" : "GET",
+        "statusErrorCode" : "ivy:error:rest:client",
+        "templateParams" : {
+          "DocumentId" : "in.documentId",
+          "FileCabinetId" : "in.fileCabinet.id"
+        },
+        "resultType" : "com.axonivy.connector.docuware.connector.DocuWareProperties",
+        "responseCode" : [
+          "import com.axonivy.connector.docuware.connector.DocuWareProperty;",
+          "import com.axonivy.connector.docuware.connector.DocuWareProperties;",
+          "",
+          "out.properties = new DocuWareProperties();",
+          "for (DocuWareProperty p : result.properties) {",
+          "  if (p.itemElementName == \"Int\" || p.itemElementName == \"Decimal\" || p.itemElementName == \"String\") {",
```

Comment:
would be nice if you switch position in this condition and rename p to property for more meaningful